getting-started/generative-ai.rst

```diff
@@ -24,3 +24,16 @@ Unacceptable uses
 Maintainers may close issues and PRs that are not useful or productive, including
 those that are fully generated by AI. If a contributor repeatedly opens unproductive
 issues or PRs, they may be blocked.
+
+Considerations or Keys for Success
+==================================
```

Comment:
Use sentence case for headers:

https://devguide.python.org/documentation/style-guide/#capitalization
```suggestion
Considerations or keys for success
==================================

```